### PR TITLE
Cnf930rerun

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
Recommitted.

### CODE REVIEW

File: modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc [hunk 1]
```diff
@@ -415,6 +418,10 @@ The DPDK related tests require:
 * `SriovNetworkNodePolicy`.
 * A node matching both the `SriovNetworkNodePolicy` and a `MachineConfig` that enables SCTP.
 
+.XT_U32 tests
+
+* A node with a Machine Config that enables XT_U32.
```
Comment (ahardin-rh) [inline modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc:423]: machine config

File: modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc [hunk 2]
```diff
@@ -591,6 +598,11 @@ Depending on the feature, running the test suite could cause different impacts o
 
 SCTP tests just run different pods on different nodes to check connectivity. The impacts on the cluster are related to running simple pods on two nodes.
 
+[id="cnf-performing-end-to-end-tests-xtu32_{context}"]
+=== XT_U32
+
+XT_U32 tests just run pods on different nodes to check iptables rule that utilize xt_u32. The impacts on the cluster are related to running simple pods on two nodes.
```
Comment (ahardin-rh) [inline modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc:604]: s/tests just run/tests run

File: modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        }
+  paused: false
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-cnf: ""
+
+EOF
+----
+
+. Include the worker node in the above machine config pool by labelling it with the `node-role.kubernetes.io/worker-cnf` label:
```
Comment (ahardin-rh) [inline modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc:84]: labeling

File: modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc (hunk 1 above)
Comment (StephenJamesSmith) [reply modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc:423]: Change made.

File: modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc (hunk 2 above)
Comment (StephenJamesSmith) [reply modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc:604]: Change made.

File: modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc (hunk 3 above)
Comment (StephenJamesSmith) [reply modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc:84]: This file is part of cnf950. I'll make this change there.

**Comment by StephenJamesSmith** [comment]
Added Ashley's comments.

**Comment by ahardin-rh** [comment]
The .github/CODEOWNERS file update that Vikram did was pulled into this PR, probably when you squashed.

**Review by codyhoag — APPROVED** [review]
Got the go ahead from @ahardin-rh for merging. Will follow up with a couple tweaks.

**Pull request merged**